packages/mobile/src/firebase/remoteConfigValuesDefaults.e2e.ts

```diff
@@ -14,8 +14,8 @@ export const REMOTE_CONFIG_VALUES_DEFAULTS: Omit<
   inviteRewardsEnabled: false,
   inviteRewardCusd: 1,
   inviteRewardWeeklyLimit: 20,
-  walletConnectV1Enabled: false,
-  walletConnectV2Enabled: false,
+  walletConnectV1Enabled: true,
+  walletConnectV2Enabled: true,
```

Comment:
just checking we want/need these changes here? i thought wallet connect v2 was disabled because it was causing issues?

Reply:
Good catch! I'll disable V2 until we enable it.